setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = dirty_cat
+version = file: dirty_cat/VERSION.txt
+description = Machine learning with dirty categories.
+long_description = file: README.rst
+author = Patricio Cerda
+author_email = patricio.cerda@inria.fr
+license = BSD
+license_files = LICENSE.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering
+    Topic :: Software Development :: Libraries
+project_urls =
+    Homepage = https://dirty-cat.github.io/
+    Source = https://github.com/dirty-cat/dirty_cat
+
+[options]
+packages = find:
+install_requires =
+    scikit-learn>=0.23.0
+    numpy>=1.17.3
+    scipy>=1.4.0
+    pandas>=1.2.0
+    requests
+    joblib
+python-requires = >=3.8
+
+[options.extras_require]
+dev =
+    pytest
+    pytest-cov
+    coverage
+    mypy
+    numpydoc
+    flake8
+doc =
+    sphinx-gallery
+    sphinxext-opengraph
+    matplotlib
+    seaborn
```

Comment:
Please add `statsmodels` here as well (added as doc dependancy with the latest PR) :)